sycl/doc/GetStartedGuide.md

```diff
@@ -278,6 +276,50 @@ command:
     c:\oclcpu_rt_<cpu_version>\install.bat c:\tbb_<tbb_version>\tbb\bin\intel64\vc14
     ```
 
+### Obtain prerequisites for ahead of time (AOT) compilation
+
+[Ahead of time compilation](CompilerAndRuntimeDesign.md#ahead-of-time-aot-compilation)
+requires OpenCL ahead of time compiler available in `PATH`. There is
+AOT compiler for each OpenCL device type (`GPU`, `CPU` and `ACC`
+(accelerator)).
+
+#### GPU
+
+* Linux
+
+  There are two ways how to obtain AOT GPU compiler `ocloc`:
+  * (Ubuntu) Download and install intel-ocloc_***.deb package from
+    [intel/compute-runtime releases](https://github.com/intel/compute-runtime/releases).
+    This package should have the same version as OpenCL GPU runtime installed
+    on the system.
+  * (other distros) `ocloc` is a part of
+    [Intel&reg; software packages for general purpose GPU capabilities](https://dgpu-docs.intel.com/index.html).
```

Comment:
Finding the ocloc under the link is challenging. Does ocloc package is available under the link?
Is it possible to provide direct link?

Reply:
No, it's not available under the link. According to the instruction, user should install intel-opencl package using yum/apt/zypper/etc, and ocloc is considered as part of this package.